### ISSUE TITLE
Filter subscriptions

### ISSUE THREAD

**Issue by GaborBesnyoi** [projectkudu/AzureResourceExplorer#323]
Hi

Is it possible or will it be possible to filter the subscription list to a desired one? Our tenant has a lot of subscriptions and we would like to have a shorter list available in the tool (to have only the core subs available) - or to have it import the filter from the Azure portal settings?

Thank you in advance.

**Comment by balag0** [comment]
Just trying to understand the motivation - Is it because the app startup is very slow trying to load all the subs?

**Comment by crower19** [comment]
no!. The problem is that by default only load a few subscriptions and we cant see the rest of subscriptions. We have more than 300 subscriptions and there isn't any way to filter in order to view that we need

**Comment by TechWatching** [comment]
I have the same exact problem. It does not load my desired subscription, I don't know why.

**Comment by balag0** [comment]
Does the subscription show up when browsing to https://resources.azure.com/api/subscriptions

**Comment by domiwar** [comment]
Same problem form me

**Comment by domiwar** [comment]
> Does the subscription show up when browsing to https://resources.azure.com/api/subscriptions

No
